cmd/lakectl/cmd/common_helpers.go

```diff
@@ -29,7 +29,7 @@ var noColorRequested = false
 // ErrInvalidValueInList is an error returned when a parameter of type list contains an empty string
 var ErrInvalidValueInList = errors.New("empty string in list")
 
-var accessKeyRegexp = regexp.MustCompile(`^AKIA[I|J][A-Z0-9]{14}Q$`)
+var accessKeyRegexp = regexp.MustCompile(`^AKIAJ[A-Z2-7]{14}Q$`)
```

Comment:
As explained at length in the summary, not a fan.  But if we're here, we mightfix the stray `|` in the regexp:
```suggestion
 var accessKeyRegexp = regexp.MustCompile(`^AKIA[IJ][A-Z0-9]{14}Q`)
```

Reply:
@arielshaqed I agree. Thanks!